week2/2-website/css/style.css

```diff
@@ -1,19 +1,261 @@
+
 }
+.devices li{
 
-/**
- * Add your custom styles below
- * 
- * Remember: 
- * - Be organised, use comments and separate your styles into meaningful chunks
- *    for example: General styles, Navigation styles, Hero styles, Footer etc.
- * 
- * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
- */
+    text-align: center;
+}
+header {
+width: 100%;
+background-color: white;
+height: 6vh;
+}
+.content-nav {
+    align-items: center;
+    height: 100%;
 
+}
+.links {
+ width: 80% ;
+ justify-content: flex-end;
+ height: 100%;
+ align-items: center;
+}
+.rapper {
+    border:1px solid green;
+    display: flex;
+    width: 70%;
+    margin:auto;
+    justify-content: space-between;
+    align-items: center;
+}
+nav{
+    position: fixed;
+    width: 100%;
+    border-bottom: 0.001rem solid black;
+    display: flex;
+    text-decoration: none;
+    border: 1px solid red;
+    justify-content: space-around;
+    }
+    nav-content {
+    display: flex;
+    justify-content: center;
+    flex-direction: row;
+    align-items: centerjustify;
+    padding: 0.5rem;
+    height: 5rem;
+    margin: 0.5rem;
+    
+    }
+    nav-list {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    list-style-type: none;
+    font-size: 1.1rem;
+    padding: 0.5rem;
+    margin:0.3rem;
+    list-style : none;
+}
+    nav a {
+        text-decoration: none;
+        color: rgb(226, 213, 213);
+    
+    }
+
+    .links {
+        list-style:none;
+        display: flex;
+        flex-direction: row;
+        align-items: flex-end;
+        
+    }
+
+.nav-link {
+
+    padding: 3rem;
+}
+    #logo {
+    height: 2.75rem;
+    width: 2.75rem;
+    margin-right: 9rem;
+    }
+    .introduction {
+        background: url(../img/first-background.jpg)no-repeat center;
+        padding: 25%;
+        color: white;
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+        justify-content: center;
+        text-align: center;
+        background-size: cover;
+    }
+    .intro-content {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    }
+    h1
+    {
+    color: black;
+    font-size: 3rem;
+    font-weight: lighter;
+    margin-top: 15rem;
+    }
+    p {
+    color: black;
+    font-size: 1.5rem;
+    font-weight: lighter;
+    }
+    button {
+    
+    text-align: center;
+    font-size: 1.8rem;
+    border-radius: 0.2rem;
+    border: none;
+    width:12rem;
+    padding:1rem;
+    margin: 2rem;
+    }
+    button:hover {
+    background-color: rgb(247, 68, 3);
+    }
+    .icons {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    }
+    .icons-content {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    }
+    h2 {
+    text-align: center;
+    font-size: 3rem;
+    font-weight: lighter;
+    margin-top: 8rem;
+    }
+    .icon-list{
```

Comment:
This would work well, but in your html this section is given the class of `icons`. Therefore the styling isn't being applied.